administrator/components/com_admin/sql/updates/mysql/4.0.0-2021-05-07.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE `#__mail_templates` SET `subject` = 'COM_PRIVACY_EMAIL_DATA_EXPORT_COMPLETED_SUBJECT' WHERE `template_id` = 'com_privacy.userdataexport';
+UPDATE `#__mail_templates` SET `body` = 'COM_PRIVACY_EMAIL_DATA_EXPORT_COMPLETED_BODY' WHERE `template_id` = 'com_privacy.userdataexport';
```

Comment:
```suggestion
UPDATE `#__mail_templates`
   SET `subject` = 'COM_PRIVACY_EMAIL_DATA_EXPORT_COMPLETED_SUBJECT',
       `body` = 'COM_PRIVACY_EMAIL_DATA_EXPORT_COMPLETED_BODY'
 WHERE `template_id` = 'com_privacy.userdataexport';
```
In opposite to `CREATE TABLE ... MODIFY column`statements, where we need to use separate statements for each column because the database checker cannot handle it for multiple column modifications in 1 statement, there is no need to split the update of one and the same record here. The database checker will still recognize that it's an update statement which is to be ignored.